app/enterprise/1.3-x/brain-immunity/install-configure.md

```diff
@@ -188,539 +200,9 @@ Requests to the status endpoint will confirm the Collector App is up and running
-
-
-
-* `base_url`: The url of the traffic used to train the model.
-* `method`: The method of the traffic used to train the model.
-* `route_id`: The Kong route_id that the traffic used to train the model is associated with. **service_id**: The Kong service_id that the traffic used to train the model is associated with. **model_version_id**: The model version number of the current, active model.
-* `active_models`: A json object containing information on the active status of each of the 6 core alert types in Immunity (unknown_parameters, abnormal_value, latency, traffic, status codes, and value_type).
-
-
-
-In this object, the value is a specific alert type and the value is a boolean value where True indicates that the model is actively monitoring for that alert type.
-
-
-In general, if a endpoint + method combination model does not appear on the returned object from `/monitoredendpoints`, this is likely because not enough traffic has been seen by Immunity to build a reliable model.
-
-### Configure Auto-Training
-
-#### Restarting Training Schedules
-
-Immunity automatically sets up training jobs when it first starts up, and retrains all models on an optimized schedule based on time since data started flowing through Immunity. If you have experienced large changes in the type of data you expect to be coming through Immunity and do not feel comfortable choosing an "optimal" time period to use for retraining with the /trainer endpoint, you can re-trigger Immunity's auto-training by posting to the /trainer/reset endpoint. Immunity will then recreate its retraining schedule as if it was just being started and newly ingesting data.
-
-```
-curl -X POST http://<COLLECTOR_HOST>:<COLLECTOR_PORT>/trainer/reset
-```
-
-#### Configuring Auto-Training Rules
-
-For best use, Immunity retrains on a regular basis. If you do not feel like you need to retrain your models regularly and are happy with the current model you have now, you can stop auto retraining via post request to the /trainer/config endpoint. This endpoint takes these parameters:
+## Everything is ready to go
+Collector App and Kong are up and ready to analyze incoming traffic for [alerts](/enterprise/{{page.kong_version}}/brain-immunity/alerts), [auto-generate specs](/enterprise/{{page.kong_version}}/brain-immunity/auto-generated-specs), and display your traffic visually in [Service-Map](/enterprise/{{page.kong_version}}/brain-immunity/service-map).
 
+For any problems encountered while setting up Collector App, Collector Plugin, or configuring other aspects of Brain and Immunity, please check out [troubleshooting](troubleshooting.md) for help debugging common problems.
```

Comment:
Looks like troubleshooting.md isn't exposed in the nav, need to add it. The link will also need to be: /enterprise/{{page.kong_version}}/brain-immunity/troubleshooting